src/icp_v2.cc

```diff
@@ -54,7 +54,7 @@ class DelayedUdpSend {
     icp_common_t *msg = nullptr; ///< ICP message with network byte order fields
     DelayedUdpSend *next = nullptr; ///< invasive FIFO queue of delayed ICP messages
     AccessLogEntryPointer ale; ///< sender's master transaction summary
-    struct timeval queue_time = {0, 0}; ///< queuing timestamp
+    struct timeval queue_time = { }; ///< queuing timestamp
```

Comment:
AFAICT, this is the only place where this PR is using a spaced version. All other places are using `{}`. While the official code is arguably inconsistent, I suggest moving towards a spaceless version for the PR consistency sake and to follow the last-resort "less ink" principle.

```suggestion
    struct timeval queue_time = {}; ///< queuing timestamp
```